stdlib/ballerina-mime/src/main/ballerina/mime/natives.bal

```diff
@@ -266,214 +266,216 @@ public type Entity object {
 
     documentation {
-        Set body parts to entity. This method will override any existing content-type headers
-        with the default content-type 'multipart/form-data'. You can override the default value by passing
-        the content-type as an optional parameter.
+        Sets body parts to entity. This method overrides any existing `content-type` headers
+        with the default content type `multipart/form-data`. The default value `multipart/form-data` can be overridden
+        by passing the content type as an optional parameter.
 
-        P{{bodyParts}} Represent the body parts that needs to be set to the entity
+        P{{bodyParts}} Represents the body parts that needs to be set to the entity
         P{{contentType}} Content-type to be used with the payload. This is an optional parameter.
-        'multipart/form-data' will be used as the default value
+                         `multipart/form-data` is used as the default value.
     }
     public native function setBodyParts(Entity[] bodyParts, string contentType = "multipart/form-data");
 
     documentation {
-        Get the header value associated with the given header name.
+        Gets the header value associated with the given header name.
 
-        P{{headerName}} Represent header name
-        R{{}} Return header value associated with the given header name as a `string`. If multiple header values are
-              present, then the first value will be returned
+        P{{headerName}} Represents header name
+        R{{}} Header value associated with the given header name as a `string`. If multiple header values are
+              present, then the first value is returned
     }
     public native function getHeader(@sensitive string headerName) returns @tainted string;
 
     documentation {
-        Get all the header values associated with the given header name.
+        Gets all the header values associated with the given header name.
 
-        P{{headerName}} Represent the header name
-        R{{}} Return all the header values associated with the given header name as a `string[]`
+        P{{headerName}} Represents the header name
+        R{{}} All the header values associated with the given header name as a `string[]`
     }
     public native function getHeaders(@sensitive string headerName) returns @tainted string[];
 
     documentation {
-        Get all header names.
+        Gets all header names.
 
-        R{{}} Return all header names as a `string[]`
+        R{{}} All header names as a `string[]`
     }
     public native function getHeaderNames() returns @tainted string[];
 
     documentation {
-        Add the given header value against the given header.
+        Adds the given header value against the given header.
 
-        P{{headerName}} Represent the header name
-        P{{headerValue}} Represent the header value to be added
+        P{{headerName}} Represents the header name
```

Comment:
Saying represents to a parameter sounds wrong. Maybe make these "Header to be added" and "Header value to be added"